Code/GraphMol/ChemTransforms/catch_tests.cpp

```diff
@@ -55,3 +55,67 @@ TEST_CASE("Github #1039", "[]") {
     CHECK(MolToSmiles(*pieces) == "*C.[1*]C(O)(F)Cl");
   }
 }
+
+TEST_CASE("molzip", "[]") {
+    SECTION("basic tests") {
+        auto a = "C[*:1]"_smiles;
+        auto b = "N[*:1]"_smiles;
+        auto mol = molzip(*a,*b);
+        CHECK(MolToSmiles(*mol) == "CN");
+    }
+    
+    {
+        auto a = "C[*]"_smiles;
+        auto b = "N[*]"_smiles;
+        MolzipParams p;
+        p.label = MolzipLabel::Isotope;
+        auto mol = molzip(*a,*b,p);
+        CHECK(MolToSmiles(*mol) == "CN");
+    }
+    
+    {
```

Comment:
I like the comprehensive chiral testing here, but you're missing a few cases:
- double bond stereochemistry: `auto a = "C/C=C/[*:1]"_smiles`
- stereochemistry in the second molecule in the zip: `auto b="[C@H](Br)([*:1])F"_smiles`

Reply:
Yep double bond stereo currently isn't working.